lua/source/path.lua

```diff
@@ -83,8 +93,9 @@ M.triggerFunction = function(_, _, _, manager)
   local stdout = vim.loop.new_pipe(false)
   local stderr = vim.loop.new_pipe(false)
   local handle, pid
-  handle, pid = vim.loop.spawn('ls', {
-    args = {path, '-A'},
+  print(path)
```

Comment:
Hey this is so nice! Can you get rid of the `print` though? I accidentally left that in and it should be remove.

Reply:
Hi, print removed.